src/loader.ts

```diff
@@ -77,6 +81,7 @@ export default class {
         const lmfile = `${this.cwd}/@types/${basename(this.config.importPath)}/lastModified`;
         if (lm !== await Bun.file(lmfile).text()) {
             await this.build();
+            await this.initConfigTypes();
```

Comment:
This should maybe be in a separate PR, it just regenerates the config types if the source has changed